src/main/java/com/pnu/ordermanagementapp/model/Client.java

```diff
@@ -1,15 +1,20 @@
 package com.pnu.ordermanagementapp.model;
 
-import lombok.Builder;
 import lombok.Data;
-
-import javax.persistence.*;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 import java.util.List;
 
 @Data
-@Builder
```

Comment:
Why removed?

Reply:
It's useless

Reply:
I don't see a big problem, we'll need it anyway later. But let it be.